presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParser.java

```diff
@@ -236,6 +236,7 @@ public void exitNonReserved(SqlBaseParser.NonReservedContext context)
             context.getParent().removeLastChild();
 
             Token token = (Token) context.getChild(0).getPayload();
+            warningConsumer.accept(new ParsingWarning("nonReserved word used", null, token.getLine(), token.getCharPositionInLine()));
```

Comment:
Why do we put it here rather than `visitNonReserved`? We don't have to overwhelm `PostProcessor`